src/Foundation/Accounts/code/Services/ContactFacetService.cs

```diff
@@ -224,18 +213,7 @@ private bool SetPhone(ContactFacetData data, XConnect.Contact contact, XConnectC
                 return false;
             }
             var phoneNumbers = contact.PhoneNumbers();
-            if (phoneNumbers == null)
```

Comment:
Same as above with `var phoneNumbers = contact.PhoneNumbers();`. Redundant?

Reply:
Yep, I fully agree. Doesn't make sense. I will just create a new facet and place it in the EmailList or Phone NumberList. Thanks for taking a look.